batchflow/once_pipeline.py

```diff
@@ -142,9 +142,16 @@ def init_model(self, mode, model_class=None, name=None, config=None):
         >>> pipeline.before
               .init_model('dynamic', MyModel, config={'input_shape': C(lambda batch: batch.images.shape[1:])})
         """
-        self.pipeline.models.init_model(mode, model_class, name, config=config)
+        self._add_action(INIT_MODEL_ID, _args=dict(mode=mode, model_class=model_class, model_name=name, config=config))
         return self
 
+    def _exec_init_model(self, action):
```

Comment:
That would break a lot of things.
Make proper tests for all modes with multiple models in the pipeline